src/main/scala/tech/cryptonomic/conseil/routes/Data.scala

```diff
@@ -56,7 +56,7 @@ class Data(config: PlatformsConfiguration, queryProtocolPlatform: DataPlatform)(
   val getRoute: Route =
     get {
       commonRoute {
-        (platform, network) =>
+        (platform, _) =>
```

Comment:
`network` not being used is a bug. We can fix it later. For now, can we ensure `network` remains here so the code is clear?